src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -896,7 +896,7 @@ public function load(string $pFilename, int $flags = 0)
                                     foreach ($commentsFile->commentList->comment as $comment) {
                                         $commentModel = $docSheet->getComment((string) $comment['ref']);
                                         if (!empty($comment['authorId'])) {
-                                            $commentModel->setAuthor($authors[$comment['authorId']]);
+                                            $commentModel->setAuthor($authors[(string) $comment['authorId']]);
```

Comment:
The autoincremented index for the standard PHP enumerated `$authors` array is an integer value, so you really should be casting to `int` here, not to `string`.